src/lib_gui/qt/element/code/QtCodeArea.cpp

```diff
@@ -12,6 +12,7 @@
 #include <QScrollBar>
 #include <QTextBlock>
 #include <QToolTip>
+#include <QtWidgets>
```

Comment:
We don't include all of QtWidgets to keep translations units smaller. Please specify each Qt header separately in alphabetical order.

Reply:
done